[2022-08-02 13:06:53,098] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.records_saving manual__2022-08-02T13:06:38.293151+00:00 [queued]>
[2022-08-02 13:06:53,110] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.records_saving manual__2022-08-02T13:06:38.293151+00:00 [queued]>
[2022-08-02 13:06:53,111] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 13:06:53,111] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-02 13:06:53,112] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 13:06:53,130] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): records_saving> on 2022-08-02 13:06:38.293151+00:00
[2022-08-02 13:06:53,135] {standard_task_runner.py:52} INFO - Started process 10767 to run task
[2022-08-02 13:06:53,138] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Demo', 'records_saving', 'manual__2022-08-02T13:06:38.293151+00:00', '--job-id', '306', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmpkz3x02cm', '--error-file', '/tmp/tmpdviqcval']
[2022-08-02 13:06:53,139] {standard_task_runner.py:80} INFO - Job 306: Subtask records_saving
[2022-08-02 13:06:53,212] {task_command.py:371} INFO - Running <TaskInstance: Demo.records_saving manual__2022-08-02T13:06:38.293151+00:00 [running]> on host 31353b71d6cb
[2022-08-02 13:06:53,301] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Demo
AIRFLOW_CTX_TASK_ID=records_saving
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T13:06:38.293151+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-02T13:06:38.293151+00:00
[2022-08-02 13:06:53,303] {python.py:173} INFO - Done. Returned value was: 0
[2022-08-02 13:06:53,334] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Demo, task_id=records_saving, execution_date=20220802T130638, start_date=20220802T130653, end_date=20220802T130653
[2022-08-02 13:06:53,390] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-02 13:06:53,455] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
